lib/nexmo_developer/app/controllers/application_controller.rb

```diff
@@ -32,6 +34,25 @@ def authenticate_admin!
     redirect_to root_path unless current_user.admin?
   end
 
+  def redirect_vonage_domain
+    return unless Rails.env.production? || Rails.env.test?
+
+    check_redirect_for(request)
+  end
+
+  def check_redirect_for(request)
+    case request.host
+    when 'developer.nexmo.com'
+      redirect_to("https://developer.vonage.com#{request.fullpath}",
+                  status: :moved_permanently) and return
+
+    when 'developer.nexmocn.com'
+      # TO-DO: LOCALE change this to point to the new domain with chinese content
+      redirect_to("https://developer.vonage.com#{request.fullpath}",
+                  status: :moved_permanently) and return
+    end
+  end
```

Comment:
Do we want to add a final catch-all redirect if none of the domains above are matched?

Reply:
What kind of `redirect_to` for a **catch-all** you have in your mind? If there is **no-match** we will just skip the   `before_action :redirect_vonage_domain`, right?
Or am I missing some edge cases here? 🤔